Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -321,12 +324,14 @@ internal void SetValue(BindableProperty property, object value, bool fromStyle)
 			SetValue(property, value, fromStyle, true);
 		}
 
-		internal void SetValueCore(BindablePropertyKey propertyKey, object value, SetValueFlags attributes = SetValueFlags.None)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SetValueCore(BindablePropertyKey propertyKey, object value, SetValueFlags attributes = SetValueFlags.None)
 		{
 			SetValueCore(propertyKey.BindableProperty, value, attributes, SetValuePrivateFlags.None);
 		}
 
-		internal void SetValueCore(BindableProperty property, object value, SetValueFlags attributes = SetValueFlags.None)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SetValueCore(BindableProperty property, object value, SetValueFlags attributes = SetValueFlags.None)
```

Comment:
I propose we do a BindableObject scrub in a subsequent PR and keep this a mechanical change. There's a lot of changes here and it'd be good to get them committed as a block with the invariant that no non-trivial control flow changes were made. Then a later investigator could look at the note and dismiss the PR as the source of whatever bug she's looking at.